### ISSUE TITLE
Get values of breakpoints through $screen object

### ISSUE THREAD

**Issue by MorevM** [reegodev/vue-screen#24]
Hi!
First of all, many thanks for your work. :)

I initialize plugin with Nuxt like this (simplified):
```
['vue-screen/nuxt', {
  phonePortrait: 0,
  phoneStandard: 375,
  phoneLandscape: 480,
  ...
  breakpointsOrder: [
    'phonePortrait',
    'phoneStandard',
    'phoneLandscape',
    ...
  ],
}],
```
I want to get access to breakpoints **values** through $screen object. Smth like:  
`this.$screen.breakpoints.phoneLandscape; // expect 480`

I already tried to pass custom property in configuration object (with object i needed) and it works, but only client-side (because setter called in initMediaQueries, which runs only in browser [link](https://github.com/reegodev/vue-screen/blob/fb15df58da169bc8260ce2597acf1c3768974e43/src/plugin.js#L227)). I need this values with SSR. 
I can ofc create external object with same params and inject it into Vue, but I think this issue in scope of this plugin.

So, what do you think about that? Can you provide this feature? How can I help? I think it will be useful not only for me.
Thanks!

**Comment by reegodev** [comment]
Hi @MorevM , when server side rendering there is no access to the window so the library can't check which breakpoints match.

You can check [this issue](https://github.com/reegodev/vue-screen/issues/14) for a better understanding.

I'm open for discussion on how to improve the library on this topic.

**Comment by reegodev** [comment]
Hi @MorevM , indeed exposing a `config` property could be useful. So in your example it could be used as `this.$screen.config.phoneLandscape`

What do you think?

**Comment by MorevM** [comment]
@reegodev yeah, its cover my case and should be fine with others, also making the plugin more extensible :)

**Comment by reegodev** [comment]
`$screen.config` is available in version 1.5.0
